src/libopensc/cwa14890.c

```diff
@@ -1340,8 +1346,8 @@ int cwa_create_secure_channel(sc_card_t * card,
 
 	/* compose signature data for external auth */
 	res = cwa_prepare_external_auth(card,
-					icc_pubkey->pkey.rsa,
-					ifd_privkey->pkey.rsa, sn_icc, sm);
+					EVP_PKEY_get0_RSA(icc_pubkey),
+					EVP_PKEY_get0_RSA(ifd_privkey), sn_icc, sm);
```

Comment:
I don't see the error. The 1.1.0 function returns a pointer to the RSA:

RSA *EVP_PKEY_get0_RSA(EVP_PKEY *pkey);

sc-ossl-compat.h has:
#define EVP_PKEY_get0_RSA(x)            (x->pkey.rsa)